torch_geometric/nn/models/dimenet.py

```diff
@@ -454,8 +454,8 @@ def reset_parameters(self):
         for interaction in self.interaction_blocks:
             interaction.reset_parameters()
 
-    @staticmethod
-    def from_qm9_pretrained(root: str, dataset: Dataset, target: int):
+    @classmethod
```

Comment:
Just curious to know, how is using classmethod different from using staticmethod here?

Reply:
I think this makes more sense here since it can naturally handle inheritance (`cls.url` except for `DimeNet.url`).